src/context/commands.rs

```diff
@@ -454,12 +454,12 @@ api! {[
             // the only CString pointers which are not freed are those of the key_specs, lets free them here.
             key_specs.into_iter().for_each(|v|{
                 if !v.notes.is_null() {
-                    unsafe{CString::from_raw(v.notes as *mut c_char)};
+                    unsafe{ drop(CString::from_raw(v.notes as *mut c_char)) };
```

Comment:
Why calling the drop inside the unsafe block?

Reply:
Moved outside.